### ISSUE TITLE
Possibility of a quick animation when un/hiding the System Tray

### ISSUE THREAD

**Issue by RavenMacDaddy** [torchgm/RoundedTB#105]
Right now when showing the system tray it just pops up when hovering the mouse in that area, which is functional and great.

What I'm wondering is if there are possibilities to implement some sort of quick animation for the event, like a quick fade in or out depending on the event that's occurring?

It would simply make it more aesthetic.

**Comment by Dom-Juan** [comment]
Agreed, a configurable hover animation and a auto animation like you pointed out would be a real cool feature, also, a anti-analysing on the edges of the taskbar would make it real cool.

**Comment by dongle-the-gadget** [comment]
> anti-analysing on the edges of the taskbar would make it real cool.

This has been elaborated as a Windows limitation, idk if there are anyway to like draw the taskbar differently and draw anti-aliasing rounded corners.

**Comment by torchgm** [comment]
A fade might be possible, but animations won't be something I focus on just yet. There's more important core functionality I need to implement first (notably a custom autohide) which might lead me down the path of adding animations.
